planning.ipynb

```diff
@@ -211,15 +211,17 @@
     "$env:PIPENV_DEFAULT_PYTHON_VERSION=#TODO\n",
     "$env:PIPENV_PYTHON=#TODO\n",
     "$env:PIPENV_VENV_IN_PROJECT=true\n",
-    "```\n"
+    "```\n",
+    "\n",
+    "Do not forget to select your interpreter (kernel) before running the other cells in the notebook!\n"
```

Comment:
The step of selecting/creating the correct kernel is indeed missing from the description.
Admittedly, I didn´t use VSCode so maybe the Jupyter extension already takes care of this, but I had to create and select a kernel for the pipenv environment I used.

Reply:
I have added that instruction in the latest commit.